README.md

```diff
@@ -218,3 +218,37 @@ We will be using loggy for application logs.
 
 * LOG_LEVEL = ""
 * LOGGLY_TOKEN = ""
+
+## Set up our To-Do app on a local minikube cluster
+
+### Run in minikube
+
+1. Spinning up a minikube cluster
+```
+Run minikube start in an admin terminal to spin up your minikube cluster
+```
+2. Deploy a Pod running the docker image of the todo-app
```

Comment:
Before this, don't forget the user needs to build & load their image:
````md
```
docker build --target production --tag todo-app:prod .
minikube image load todo-app:prod
```
````